watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/dye_desalination_withRO.py

```diff
@@ -573,54 +589,210 @@ def initialize_costing(m):
 
 
 def display_results(m):
-    print("Unit models:")
+    print("\nUnit models:")
     m.fs.pretreatment.wwtp.report()
     m.fs.dye_separation.P1.report()
     m.fs.dye_separation.nanofiltration.report()
-    m.fs.desalination.S1.report()
-    m.fs.desalination.P2.report()
-    m.fs.desalination.P3.report()
-    m.fs.desalination.M1.report()
     m.fs.desalination.RO.report()
-    m.fs.desalination.PXR.report()
 
-    print("Streams:")
-    flow_list = ["feed", "wwt_retentate", "dye_retentate", "permeate", "brine"]
+    print("\nStreams:")
```

Comment:
For reference, there are tools in IDAES to build stream tables that could be used here.